nucypher/cli/actions/select.py

```diff
@@ -72,13 +67,6 @@ def select_client_account(
         testnet = domain != TACoDomain.MAINNET.name
```

Comment:
I know this is outside of the PR code, but we should be careful here if domain is `None`. If it is `None`, we assume testnet, is that desired?

Reply:
🤔 I'll address that in #3315

Reply:
✔️

Reply:
Was that commit pushed? Can't find it - https://github.com/nucypher/nucypher/commit/2ff02b23368b0ae84c2ad7d5cb1796ee00956b5f

Reply:
Try this one: 6d5ced138f28e39f9329592e39a2d02830f5928b

Reply:
Still not recognized as being part of any branch/pr... - 
<img width="1487" alt="Screenshot 2023-10-30 at 4 10 34 PM" src="https://github.com/nucypher/nucypher/assets/19150641/b4c81523-95bd-466c-8f75-c0cdaf05a4e1">

Reply:
Sure, that's because it's in an unmerged PR -- is there some other way to link it?

Reply:
> 🤔 ~~I'll address that in https://github.com/nucypher/nucypher/pull/3315~~ handled in https://github.com/nucypher/nucypher/commit/2ff02b23368b0ae84c2ad7d5cb1796ee00956b5f

Oh no worries, I read your scratched off comment above as no longer being addressed in 3315, and assumed it was handled in this PR. All good then